### ISSUE TITLE
Problems with full screen view

### ISSUE THREAD

**Issue by Fassi96** [TeamNewPipe/NewPipe#8608]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.23.1

### Steps to reproduce the bug

Play a video. 
Rotate the phone for full screen view.

### Expected behavior

That the video is played in full screen

### Actual behavior

The video often does not play correctly in full screen.

### Screenshots/Screen recordings

https://netherghast-my.sharepoint.com/:f:/g/personal/matthias_hanisch_netherghast_de/Eo59CbHzsnpDqLp_cYDElDABRVUSnTwrC1u553tWRqCDuw?e=lX0ZRW

### Affected Android/Custom ROM version

Android 12

### Affected device model

Samsung Galaxy Note 20 5G Ultra SM-N986B/DS

**Comment by AudricV** [comment]
> - [x] I made sure that there are no existing issues - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.

Duplicate of #7262. Please search (better, with different keywords?) for duplicates the next time you want to open an issue. Thank you.

**Comment by Fassi96** [comment]
I created the issue because the problem isn't solved yet and the ticket you linked to is already marked as solved.

**Comment by AudricV** [comment]
> the ticket you linked to is already marked as solved

Not at all: it would be closed if it was already fixed. That's not the case here. And the pull request linked is an experiment.
